src/overlays/actors/ovl_Item_Ocarina/z_item_ocarina.c

```diff
@@ -76,7 +78,8 @@ void ItemOcarina_Init(Actor* thisx, PlayState* play) {
 void ItemOcarina_Destroy(Actor* thisx, PlayState* play) {
 }
 
-void ItemOcarina_Fly(ItemOcarina* this, PlayState* play) {
+//ocarina after being thrown in the escape cutscene and beginning to fall
```

Comment:
Please polish the capitalization/punctuation
Also have a space after the `//` (or run the formatter: `./format.py` at the repo's root)